api-geotrack/src/main/java/com/geotrack/apigeotrack/controllers/GeometryController.java

```diff
@@ -20,10 +22,25 @@ public class GeometryController {
     GeometryService geometryService;
 
     // method to insert geometry zones
+    @Operation(summary = "Adiciona uma Zona Geométrica ao Banco de dados")
     @PostMapping("/add")
-    public ResponseEntity<String> insertPolygon(@RequestBody GeometryZoneRequestDTO geometryZoneRequestDTO) {
+    public ResponseEntity<String> insertGeometryZones(@RequestBody GeometryZoneRequestDTO geometryZoneRequestDTO) {
+        geometryService.insertGeometryZones(geometryZoneRequestDTO);
+        return new ResponseEntity<>("Zona inserida com Sucesso", HttpStatus.CREATED);
+    }
+
+    @Operation(summary = "Listagem de Zonas Geométricas", description = "Retorna uma lista de Zonas Geométricas ativas no banco de dados")
+    @GetMapping("list")
+    public ResponseEntity<List<GeometryZoneResponseDTO>> getAllZones() {
+        geometryService.getAll();
```

Comment:
Alterar o nome do método para quem for usar saber que o getAll é referente a zonas.